doc/mgmt/SONiC_OC_Vlan_HLD.md

```diff
@@ -0,0 +1,302 @@
+# VLAN
+OpenConfig support for VLAN interfaces
+# High Level Design Document
+#### Rev 0.1
+
```

Comment:
does this also cover IP address configuration on Vlan? if not could we share which PR we tracking that?

Reply:
IP address config, and no command will be updated in the coming HLD update.

Reply:
@justinejose91 - Please update HLD with IP address config.